affiliation/service.go

```diff
@@ -123,6 +123,8 @@ type Service interface {
 	isPrecacheRunning() bool
 	setPrecacheRunning()
 	clearPrecacheRunning()
+	IsProjectSkipped(string) bool
+	SkipDisabledProjects(project string) string
```

Comment:
nit: better to use a `pointer receiver` here.

Reply:
this string is going to be small, & perhaps, changing it to pointer would make it mutable, moreover, still we can change it, but the callers of these functions are too many to update for this trivial change. & this particular change is already merged to master branch, check it here: https://github.com/LF-Engineering/dev-analytics-affiliation/pull/123

Reply:
we can revisit this, when we update this again.